### DISCUSSION TITLE
Need help with Hyperion screen grabber (windows)

### DISCUSSION THREAD

**Discussion by Wickedv** [awawa-dev/HyperHDR#186] [General]
Hello.

so I didnt configure anything yet, but I know that if I dont want to use an HDMI capture card, I need to use the Hyperion screen grabber app.

My only problem is that my life preview is a mess.
![image](https://user-images.githubusercontent.com/67589143/148646688-225daed9-948b-473a-a5d2-475eb27bc906.png)

How can I fix this?

Thanks in advance for your help

**Comment by awawa-dev** [comment]
Hi
And why you don't want to use built-in HyperHDR's software screen-grabber that is using newer DirectX11 instead? It's different one that Hyperion is using (DirectX9) so 'Hyperion screen grabber' is not necessary anymore.

**Reply by Wickedv** [reply]
i am using screen-grabber on pc and forwarding to pi

**Reply by awawa-dev** [reply]
OK, you can do it with HyperHDR also. Unfortunately can't help with 'Hyperion screen grabber' but I guess it's probably a configuration problem when captured screen size is reduced too much in that application.

**Reply by Wickedv** [reply]
can you help me with hyperHdr then. Thanks in advance

**Reply by awawa-dev** [reply]
If you don't want to tune-up 'Hyperion screen grabber' configuration, then on Windows run HyperHDR, enable software screen capture  and forwarding to HyperHDR on the Rpi using its flatbuffer server.

**Reply by awawa-dev** [reply]
https://github.com/awawa-dev/HyperHDR/discussions/149

**Reply by Wickedv** [reply]
thanks
